tests/neg/i12573.check

```diff
@@ -1,9 +1,9 @@
 -- [E008] Not Found Error: tests/neg/i12573.scala:23:38 ----------------------------------------------------------------
 23 |val w: Value[8] = DFBits(Value[8](8)).getDFType.width // error
    |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-   |         value getDFType is not a member of DFBits[(8 : Int)].
-   |         Extension methods were tried, but the search failed with:
+   |value getDFType is not a member of DFBits[(8 : Int)].
+   |Extension methods were tried, but the search failed with:
    |
-   |             method getDFType cannot be accessed as a member of DFType.type from package object i12573$package.
-   |              Access to protected method getDFType not permitted because enclosing package object i12573$package
-   |              is not a subclass of object DFType where target is defined
+   |    method getDFType cannot be accessed as a member of DFType.type from top-level definition in package <empty>.
+   |     Access to protected method getDFType not permitted because enclosing top-level definition in package <empty>
+   |     is not a subclass of object DFType where target is defined
```

Comment:
It should be "from the top-level definitions" since we're talking about the object here, but also "top-level definitions is not a subclass of ..." isn't really meaningful, the explanation could be simplified to just say "Protected method getDFType can only be accessed from object DFType or one of its subclasses" (and the "or one of its subclasses" part could be dropped for objects too)

Reply:
Changed